src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -769,16 +806,45 @@ TEST_CASE("request_rate_multiple")
 ///
 TEST_CASE("request_rate_sequence")
 {
-  PerfAnalyzerParameters params = TestLoadManagerBase::GetSequenceTestParams();
+  PerfAnalyzerParameters params;
+  bool verify_seq_balance = false;
   bool is_sequence_model = true;
-  TestRequestRateManager trrm(params, is_sequence_model);
 
+  const auto& ParameterizeDistribution{[&]() {
+    SUBCASE("Constant") { params.request_distribution = CONSTANT; }
+    SUBCASE("Poisson") { params.request_distribution = POISSON; }
+  }};
+
+  SUBCASE("short running sequences")
+  {
+    params = TestLoadManagerBase::GetSequenceTestParams();
```

Comment:
Instead of piggy-backing off of this old test (which already does a ton of combinations) can we create a new test? We can still leverage the main TestSequences() function. In theory a test should only test "one thing" and at this point this test is covering the entire kitchen sink of sequences.

Reply:
refactored into 2 tests